test/core.vwtest.json

```diff
@@ -5464,5 +5450,59 @@
     "input_files": [
       "train-sets/explore_eval_egreedy.dat"
     ]
+  },
+  {
+    "id": 423,
+    "desc": "Spin off automl model to lower bitsize to cb_explore",
+    "vw_command": "--cb_explore_adf -d train-sets/cb_test.ldf --noconstant -f models/aml_spin_off.m -b 18 --predict_only_model --automl 2",
+    "diff_files": {
+      "stderr": "train-sets/ref/spin_off_aml_save.stderr"
+    },
+    "input_files": [
+      "train-sets/cb_test.ldf"
+    ]
+  },
+  {
+    "id": 424,
+    "desc": "Load spun off automl model and output invert hash",
+    "vw_command": "--cb_explore_adf -d train-sets/cb_test.ldf --noconstant -i models/aml_spin_off.m -b 17 -q :: --invert_hash aml_spinoff.inv --predict_only_model",
+    "diff_files": {
+      "stderr": "train-sets/ref/spin_off_aml_load.stderr",
+      "aml_spinoff.inv": "pred-sets/ref/aml_spinoff.inv"
+    },
+    "input_files": [
+      "train-sets/cb_test.ldf",
+      "models/aml_spin_off.m"
+    ],
+    "depends_on": [
+      423
+    ]
+  },
+  {
+    "id": 425,
+    "desc": "Generate similar model to spun off automl using standard cb_explore_adf",
+    "vw_command": "--cb_explore_adf -d train-sets/cb_test.ldf --noconstant -f models/aml_spin_off_cb.m -b 17 --predict_only_model -q ::",
+    "diff_files": {
+      "stderr": "train-sets/ref/cb_similar_aml_spinoff_save.stderr"
+    },
+    "input_files": [
+      "train-sets/cb_test.ldf"
+    ]
+  },
+  {
+    "id": 426,
```

Comment:
you can also recycle this unit test: https://github.com/VowpalWabbit/vowpal_wabbit/blob/417012d772bf26cecc8d234a820994aa65144d82/test/unit_test/automl_weights_test.cc#L263

line 280 model is -b 20, line 279 is -b 18, so if you run --predict_only_model on 280 line one it should be equal to the other one 279 -- as in exactly the same bit by bit.

Reply:
made a test `automl_equal_spin_off_model` which is similar and shows the weights matching